core/embed/trezorhal/stm32f4/sdram.h

```diff
@@ -25,7 +25,7 @@
 extern "C" {
 #endif
 
-#include STM32_HAL_H
+#include <trezor_bsp.h>
```

Comment:
This file should probably be split into api header file and internal header (or move the internal definitions into the C file). This file is used directly in boardloader main but is in the F4 folder, so that is wrong too. Maybe we don't need to fix this here but should be done with the drivers-folders restructuring.

Reply:
Yes, I agree and created #4307.